python/cugraph/dask/pagerank/pagerank.py

```diff
@@ -278,3 +282,32 @@ def get_chunksize(input_path):
         size = [os.path.getsize(_file) for _file in input_files]
         chunksize = max(size)
     return chunksize
+
+
+def _read_csv(input_files, delimiter='\t', names=['src', 'dst'],
```

Comment:
should `delimiter` be a parameter in `read_split_csv`?

Reply:
added delimiter, names and dtype as parameters user can control